core/domain/prod_validators_test.py

```diff
@@ -6506,51 +6520,6 @@ def test_model_with_invalid_target_type(self):
             self.run_job_and_check_output(
                 expected_output, sort=True, literal_eval=False)
 
-    def test_validate_score_category_for_content_suggestion(self):
```

Comment:
Is there some way to link this with the actual domain object validation? Because I remember we removed the validation on the domain object a while back, but looks like the prod validation wasn't updated. Am I right?